nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+
+const reservations = require("../data/reservations.json");
+
+router.get("/", async (request, response) => {
+    response.send(reservations);
+  try {
+    console.log(reservations);
+    console.log("in /api/reservations");
+  } catch (error) {
+    throw error;
+  }
+});
```

Comment:
You should avoid leaving out console.log statements! They are useful while you are developing your app, but it can be confusing to have them when your app is live. Imagine a ton of messages being printed out all the time!
Also, I like that you add a try catch block, this means ensures that if an error occurs you control what the user will see :+1: Consider though, to include the response.send inside the try block instead of the console.log statements.